### ISSUE TITLE
HDD games in folder form fail to detect any DLC

### ISSUE THREAD

**Issue by Rex000** [RPCS3/rpcs3#13399]
### Quick summary

Title

### Details

The HDD games that are in folder form are not showing any DLC (PKG+RAP) that is installed in the emulator.

Extracting the DLC into the game folder results in RPCS3 not even detecting the game itself.

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/10730757/RPCS3.log.gz)

**Comment by Megamouse** [comment]
What did you expect when you change the folder of your game?
You have to install the dlc in your special folder as well

**Comment by Rex000** [comment]
You're right, it was the first thing that I did, but after that RPCS3 failed to detect the entire game altogether.

**Comment by Rex000** [comment]
You're right, it was supposed to be this way and was the first thing that I tried. I unpacked the game as well as its DLC all in one folder, but after doing that RPCS3 failed to detect the entire game altogether.

**Comment by Megamouse** [comment]
What does "Failed to detect" mean?

**Comment by Rex000** [comment]
The game wasn't appearing in the game list. I tried adding the game again multiple times, even closed and reopened the emulator, but it still wasn't appearing.

However, I think I've figured it out now. After extracting the game+updates in a folder, I extracted the DLC in it as well. During testing it came to my attention that the DLC had its own unique PARAM.SFO file that was overwriting the main game's PARAM.SFO file, and this was actually the thing that was causing the issue of the game disappearing from the game list. I don't know why that's the case. But anyway when I copied the main game's PARAM.SFO file back into the folder, the game appeared back again in the list. Seems to work fine so far and I hope it stays that way.

I wonder though, what makes the DLC for disc games work flawlessly in a different folder than the game itself? Because that was actually the reason for opening this issue.

**Comment by Megamouse** [comment]
you should install the game and dlc in rpcs3.
then move the folder to your game folder....

**Comment by Rex000** [comment]
Yep, I've been doing exactly that since I discovered the PARAM.SFO issue, just to be on the safe side.

**Comment by Rex000** [comment]
Yep, I've been doing exactly that since I discovered the PARAM.SFO issue, just to be on the safe side.

What's this info that you need?

**Comment by Megamouse** [comment]
The info would be a log of you trying to add the game to the list after you properly installed all the updates and dlcs **using rpcs3** and then moving the game to your other folder.

**Comment by Rex000** [comment]
As I mentioned, it works fine that way and the game appears in the list when added. It's only when I manually extract everything in one folder that the PARAM.SFO of the main game gets overwritten and hence the game doesn't appear in the list.

The issue as I explained is when a HDD game in folder form is kept outside of the RPCS3 folder while the DLC is installed in RPCS3, the said DLC does not get added to the game. 

Here is a log for a game that I installed in RPCS3 then moved its folder in another location. The DLC(installed in RPCS3) doesn't appear in the game:
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/10777592/RPCS3.log.gz)
